### DISCUSSION TITLE
Updated restore/prune command question

### DISCUSSION THREAD

**Discussion by FuzzyMistborn** [cupcakearmy/autorestic#132] [Q&A] [answered]
Trying to understand the breaking change for `prune` and `restore`.

"The way snapshots are referenced in the restore and prune commands has been changed. Before they were referenced by the path. Now every backup is tagged and those tags are then referenced in the cli. This means that when running restore and forget commands old backups are not taken into account anymore."

So if I have existing backups, do I have to manually rotate them out?  Or can I keep doing `autorestic forget -a -- prune`?  And for restores, does `autorestic restore -f -l docker --from b2_docker --to /home/fuzzy/docker` not restore the latest/last backup?

**Comment by cupcakearmy** [comment]
Yes, they are not taken into account, which will not delete/prune them. Restoring from the older ones should work if giving autorestic the snapshot id. Otherwise i will take the latest "new" backup.

**Reply by FuzzyMistborn** [reply]
....That's a pretty major change.  If I used `exec` to basically use restic would it work?  I just don't want to have to manually delete old snapshots for the next few months (i keep 3 months of snapshots) or wipe all my backups.

**Comment by FuzzyMistborn** [comment, marked as answer]
To answer my own question, yes you can use `exec` and define the policy manually.

`autorestic exec -av -- forget --keep-daily 1 --keep-monthly 4 --keep-weekly 8  --prune` for example.

**Reply by cupcakearmy** [reply]
I'm sry, forgot to reply.
Yes, you can do that, as exec forwarding the plain command to `resitc`
